app/models/event.rb

```diff
@@ -95,4 +87,40 @@ def fetch_github_stats
   def delete_logo
     logo.clear if logo_delete == '1'
   end
+
+  def fetch_attendee_github_stats
+    stats = {}
+
+    event_registrations.find_in_batches(batch_size: 5) do |er_batch|
+      er_batch.each do |er|
+        begin
+          stats.merge!(er.user.email => er.fetch_github_stats)
+        rescue
```

Comment:
I was having the rescue area print details about the call that could be dug into later, but I removed it. Should I put it back?

It would require running requests again with the same user / project to get more details and also printing the GitHub API request, but that could be helpful to do at some point to figure out why things are failing.